static/js/time-extend.js

```diff
@@ -1,23 +1,29 @@
 add_local_tz = (selector) => {
   const regex_time = new RegExp("\\((.*)-(.*) (.*)\\)");
   const guess_tz = moment.tz.guess(true);
+  const utc_offsets = { UTC: 0, GMT: 0, PDT: -7 };
 
   $(selector).each(function () {
     const t = $(this).text();
     const res = regex_time.exec(t);
-    if (res) {
-      const start_time = moment.utc(`2020-07-05 ${res[1]}`);
-      const end_time = moment.utc(`2020-07-05 ${res[2]}`);
-      const local_start = start_time.tz(guess_tz);
-      const local_start_and_tz = start_time.format("HH:mm");
-      const local_end = end_time.tz(guess_tz);
+    if (res && res[3] in utc_offsets) {
+      const utc_offset = utc_offsets[res[3]];
+      const start_time = moment
+        .utc(`2020-07-05 ${res[1]}`)
+        .utcOffset(utc_offset, true);
+      const end_time = moment
+        .utc(`2020-07-05 ${res[2]}`)
+        .utcOffset(utc_offset, true);
+      const local_start = moment(start_time).tz(guess_tz);
+      const local_start_and_tz = local_start.format("HH:mm");
+      const local_end = moment(end_time).tz(guess_tz);
```

Comment:
`.tz(...)` mutates `end_time`, so we should clone it first. Means we don't need the `.utc()` call below.

Reply:
Can you add this change to a new PR?
In #512, I have changed the socials.yml and site_data.py so that we can re-use the `SessionInfo` class as an universal way to display session time. Therefore, we don't need to deal with the `utc_offsets` here -- all time by default should be GMT.

Reply:
@GuyAglionby ^